### PR TITLE
Dark-mode issues fixed.

### PR DESCRIPTION
**Pull request by mohit-003**
**Dark-mode issues fixed.**
for issue #7 #133 and [GCI task](https://codein.withgoogle.com/dashboard/task-instances/6451111322451968/)

### **Interactive logic gates**: dotted order colour changed to grey,and backgroung for gates is enabled

In Boolean-Function the text style was different than other pages so it is made uniform.
The h1 tag's style is removed as it made it black color and it was not visible in Dark-Mode,
The p tag's style is removed as it made the text color different from other pages.

### CODE REVIEW

**Comment by mohit-003** [comment]
@aayush-05 please review my task.

**Comment by aayush-05** [comment]
Great work @mohit-003 
Two things to be done :
Remove this particular change
image
Also, is there any particular reason for which you changed the font style. If not, please revert it back to make the look on all the pages uniform.
image

**Comment by aayush-05** [comment]
Also, I could find the changes in the code for this :
image

And you haven't mentioned about this change in the pr description :
image
(Please provide required screenshots for it)

**Comment by mohit-003** [comment]
> Great work @mohit-003
> Two things to be done :
> Remove this particular change
> image
> Also, is there any particular reason for which you changed the font style. If not, please revert it back to make the look on all the pages uniform.
> image

In Boolean-Function the text style was different than other pages so I made it uniform.

I removed the <h1> 's styles as it made it black color and it was not visible in Dark-Mode,
The <p> 's style was removed as it made the text color different from other pages.

File: _sass/base.scss [hunk 1]
```diff
@@ -108,3 +108,16 @@ hr {
   background-color: $border-color;
   border: 0;
 }
+
+select,
+option {
+  color: $black;
```
Comment (mohit-003) [inline _sass/base.scss]: For the drop box text I have made the changes here

**Comment by mohit-003** [comment]
Yep done @aayush-05

**Comment by aayush-05** [comment]
Isn't all the pages getting the font style from the base file. So rather removing the font family attribute would bring in more uniformity. Try that @mohit-003

**Comment by mohit-003** [comment]
Please review it now @aayush-05

**Comment by aayush-05** [comment]
Hey @mohit-003 
I noticed you added the whole code for the sidebar in the kmap.html file to add the sidebar in the Interactive K Map section.
Try changing the layout for kmapi.html page to default, rather than editing the kmap layout file.

**Comment by aayush-05** [comment]
Looks good to me, have a look @tachyons @ankitkataria

File: docs/registers/sp.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: default
 title: Serial Input Parallel Output
+comments: true
```
Comment (tachyons) [inline docs/registers/sp.md]: intentional?

File: docs/registers/ss.md [hunk 3]
```diff
@@ -50,3 +51,4 @@ Similarly with Din = 1 and with the fourth negative clock edge arriving, the sto
 ## Waveforms
 <div style="text-align:center"><img src="../../assets/images/siso_waveform.jpg" /></div>
 
+{% include disqus.html %}
```
Comment (tachyons) [inline docs/registers/ss.md]: This seems unrelated to darkmode

**Comment by aayush-05** [comment]
Disqus comment section wasn't visible on some pages, so the student added code for that as well. Should that be a separate pr ?

**Comment by Ananthavijay** [comment]
I think there is a beginner task related to the comment section

**Comment by mohit-003** [comment]
Should I make a separate PR then?

**Comment by tachyons** [comment]
Those should be in separate PR

**Comment by mohit-003** [comment]
Yep done! @aayush-05  @tachyons

**Comment by aayush-05** [comment]
Why is this change required ?
image

**Comment by mohit-003** [comment]
> Why is this change required ?
> image

Yes it is required because I changed the layout to default from kmap. So without the scrollcount the the kmapi won't give and outputs.

**Comment by aayush-05** [comment]
@tachyons clear from my side, even the comment section thing has been removed now. 
Ready to approve from your side ?

**Comment by aayush-05** [comment]
@mohit-003 would you please fix the conflicts ?

**Comment by mohit-003** [comment]
Yes I'll do it now

**Comment by mohit-003** [comment]
@aayush-05 I'll make a new PR as this branch got deleted.
